js/views/modals/About/Donations.js

```diff
@@ -0,0 +1,50 @@
+import loadTemplate from '../../../utils/loadTemplate';
+import baseVw from '../../baseVw';
+import { clipboard } from 'electron';
+import QR from 'qr-encode';
+
+const qr = QR;
```

Comment:
Why make this assignment instead of just using the `QR` form the import? Essentially, what you are doing is not much different from:

```
const QR = 'hello';
const qr = QR;
```